addons/l10n_eu_service/tests/test_oss.py

```diff
@@ -0,0 +1,99 @@
+        self.assertTrue(tag_base_exempted_49 in refund_tax_tag_ids, "Refund tag from Belgium CoA not correctly linked")
+
+@tagged('post_install', '-at_install')
+class TestOSSUSA(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(self, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        self.company_data['company'].country_id = self.env.ref('base.us')
+        self.company_data['company']._map_eu_taxes()
+
+    def test_no_oss_tax(self):
+        # get an eu country which isn't the current one:
+        another_eu_country_code = (self.env.ref('base.europe').country_ids - self.company_data['company'].country_id)[0].code
+        tax_oss = self.env['account.tax'].search([('name', 'ilike', f'%{another_eu_country_code}%')], limit=1)
+        self.assertTrue(len(tax_oss) == 0, "OSS tax shouldn't be instanced on a US company")
+
+class TestOSSMap(AccountTestInvoicingCommon):
+
+    def test_oss_map(self):
+        # For each country in base.eu,
+        # if it's template is installed,
+        # check if the xml_id referenced in eu_tag_map is correct
+        country_to_chart_template_map = {
+            'AT': 'l10n_at.l10n_at_chart_template',
+            'BE': 'l10n_be.l10nbe_chart_template',
+            'BG': '',
+            'CY': '',
+            'CZ': '',
+            'DE': '',
+            'DK': 'l10n_dk.dk_chart_template',
+            'EE': '',
+            'ES': 'l10n_es.account_chart_template_common',
+            'FI': 'l10n_fi.fi_chart_template',
+            'FR': 'l10n_fr.l10n_fr_pcg_chart_template',
+            'GR': 'l10n_gr.l10n_gr_chart_template',
+            'HR': "l10n_hr.l10n_hr_chart_template_rrif",
+            'HU': 'l10n_hr.hungarian_chart_template',
+            'IE': 'l10n_ie.l10n_ie',
+            'IT': 'l10n_it.l10n_it_chart_template_generic',
+            'LT': 'l10n_lt.account_chart_template_lithuania',
+            'LU': 'l10n_lu.lu_2011_chart_1',
+            'LV': '',
+            'MT': '',
+            'NL': 'l10n_nl.l10nnl_chart_template',
+            'PL': 'l10n_pl.pl_chart_template',
+            'PT': 'l10n_pt.pt_chart_template',
+            'RO': 'l10n_ro.ro_chart_template',
+            'SE': 'l10n_se.l10nse_chart_template',
+            'SI': 'l10n_si.gd_chart',
+            'SK': '',
+        }
+        for country, chart_template_xml_id in filter(lambda d: d[1], country_to_chart_template_map.items()):
+            try:
+                self.env.ref(chart_template_xml_id)
+            except ValueError:
+                # the chart template isn't in DB (or its xml_id is wrong)
+                continue
+
+            oss_tags = EU_TAG_MAP.get(country)
+            for xml_id in filter(lambda d: d, oss_tags.values()):
+                try:
+                    self.env.ref(xml_id)
+                except ValueError as e:
+                    e.args = (f"xml_id {xml_id} from EU_TAG_MAP is wrong", *e.args)
+                    raise
```

Comment:
Discussed with WAN:
This part could be better by adding the optional parameter to ref to avoid raising the error.
Then just assert the tax exist.

Additional idea: collect the wrong xml_id and assert there are none of them.